doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,104 @@ The query returns all the entities that match all the matching conditions (AND l
-      Ranges can only be used with attributes that represent dates (in ISO8601 format), numbers or
-      strings.
+      For an entity to match, it must contain the target property (temperature),
+      and its value must be in the interval between the minimum and maximum of the range (both included).
+      Ranges can only be used with target properties that represent dates (in ISO8601 format),
+      numbers or strings.
 + **Unequal**: `!=`. This operator accepts the following types of right-hand side:
-    + Single element, e.g. `temperature!=41`. For an entity to match, it must contain an attribute
-      with that exact name (temperature) and whose value is **not** that exact value (41).
+    + Single element, e.g. `temperature!=41`. For an entity to match, it must contain the target
+      property (temperature) and its value must **not** be the query value (41).
     + A list of comma-separated values, e.g. `color!=black,red`. For an entity to match, it must
-      contain an attribute with that name, and whose value is **not** any of the values in the list
-      (AND clause).
+      contain the target property, and its value is **not** any of the values
+      in the list (AND clause) (or not include **any** of the values in the list in the case the target
+      property value is an array).
       Eg. entities whose attribute `color` is set to `black` will not match, while entities whose
       attribute `color` is set to `white` will match.
     + A range, specified as a minimum and maximum separated by `..`, e.g. `temperature!=10..20`.
-      For an entity to match, it must contain an attribute with that exact name (temperature) and
-      whose value is **not** in the interval between the minimum and the maximum (both included).
-      Ranges can only be used with attributes that represent dates (in ISO8601 format), numbers or
-      strings.
+      For an entity to match, it must contain target property (temperature) and its value must **not**
+      be in the interval between the minimum and the maximum (both included). Ranges can only be used
+      with elements target properties that represent dates (in ISO8601 format), numbers or strings.
 + **Greater than**: `>`. The right-hand side must be a single element, e.g. `temperature>42`.
-    For an entity to match, it must contain an attribute with that exact name (temperature) and
-    whose value is strictly greater than that value (42).
-    This operation is only valid for attributes of type date, numbers or strings (used with
-    attributes of other types may lead to unpredictable results).
+    For an entity to match, it must contain the target property (temperature) and
+    its value must be strictly greater than that query value (42).
+    This operation is only valid for target properties of type date, numbers or strings (used with
+    target properties of other types may lead to unpredictable results).
```

Comment:
unpredictable results ...
Don't we return an error when invalid types are found?